examples/python/hello_migraphx.py

```diff
@@ -0,0 +1,253 @@
+def preprocess(img_data):
+    # todo: are these vectors based on RGB images or BGR?  Results seem good
+    mean_vec = np.array([0.485, 0.456, 0.406])
+    stddev_vec = np.array([0.229, 0.224, 0.225])
+    norm_img_data = np.zeros(img_data.shape).astype('float32')
+    for i in range(img_data.shape[0]):  
+        norm_img_data[i,:,:] = (img_data[i,:,:]/255 - mean_vec[i]) / stddev_vec[i]
+    return norm_img_data
+
+#
+# Read command line arguments
+#
+root = os.getenv("PROTEUS_ROOT")
+assert root is not None
+
+# Get the arguments required from the user
+parser = argparse.ArgumentParser(
+    description="Example client Proteus Migraphx worker"
+)
+
+parser.add_argument(
+    "--modelfile",
+    "-m",
+    type=str,
+    required=False,
+    default= os.path.join(root,  r"external/artifacts/migraphx/resnet50-v1-7/resnet50-v1-7.onnx"),
+    help="Location of model file on server",
+)
+
+parser.add_argument(
+    "--labels",
+    "-l",
+    type=str,
+    required=False,
+    default= os.path.join(root,  r"external/artifacts/migraphx/resnet50-v1-7/imagenet_simple_labels.json"),
+    help="The file containing label names for the model's categories",
+)
+
+parser.add_argument(
+    "--image",
+    "-i",
+    type=str,
+    required=False,
+    default= os.path.join(root,  r"tests/assets/girl-1867092_640.jpg"),       # a girl in a sweatshirt
+    help="An image to try inference on.  Use git-lfs to pull image assets",
+)
+
+parser.add_argument(
+    "--image2",
+    "-g",
+    type=str,
+    required=False,
+    default= os.path.join(root,  r"tests/assets/bicycle-384566_640.jpg"),       # a bicyclist
+    help="A second image to try inference on",
+)
+
+# Parse arguments
+args = parser.parse_args()
+modelname = args.modelfile
+imagename = args.image
+imagename2 = args.image2
+labels_file = args.labels
+
+#
+#           End read command line arguments
+#
+
+#  load the onnx model to find the input shape, see https://stackoverflow.com/questions/56734576/find-input-shape-from-onnx-file
+#  We assume here that client is on the same file system as the server
+#  This code is applicable to any onnx model, but for resnet50 the required shape could have been hardcoded:   [1, 3, 224, 224]
+shape=[]
+model = onnx.load(modelname)
+for input in model.graph.input:
+    if input.name == 'data':
+        print (input.name, end=": ")
+        # get type of input tensor
+        tensor_type = input.type.tensor_type
+        # check if it has a shape:
+        if (tensor_type.HasField("shape")):
+            # iterate through dimensions of the shape:
+            for d in tensor_type.shape.dim:
+                # the dimension may have a definite (integer) value or a symbolic identifier or neither:
+                if (d.HasField("dim_value")):
+                    shape.append(d.dim_value)
+        else:
+            print ("unknown rank", end="")
+        print()
+        break
+
+print('This model\'s shape of input image is ', shape)    

```

Comment:
Run `black .` in the repository root to format Python code

Reply:
fixed